test/e2e/framework/util.go

```diff
@@ -376,10 +376,10 @@ func SkipIfMissingResource(clientPool dynamic.ClientPool, gvr schema.GroupVersio
 }
 
 // ProvidersWithSSH are those providers where each node is accessible with SSH
-var ProvidersWithSSH = []string{"gce", "gke", "aws"}
+var ProvidersWithSSH = []string{"gce", "gke", "aws", "local"}
```

Comment:
you sure? I see several `framework.SkipUnlessProviderIs(framework.ProvidersWithSSH...)` calls still.